src/components/editor/EditorLogin.jsx

```diff
@@ -51,6 +85,29 @@ export default class Login extends BaseComponent {
     }
   }
 
+  onGoogleResponse(response) {
+    if (response.error) {
+      alert(`Google Login Failed.\n${response.error}`);
+      return;
+    }
+
+    const token = response.tokenId;
+
+    const xhr = new XMLHttpRequest();
+    xhr.open('POST', '/googlelogin', false);
+    xhr.setRequestHeader('Content-type', 'application/x-www-form-urlencoded');
```

Comment:
Hmm... I'm pretty sure and also just ecosia'd it that the correct HTTP header is called `Content-Type`, I'm not quite sure how strict different libraries are about it but let's just stick to the standards

Reply:
ohh that was a typo, will fix